pycrosskit/shortcuts.py

```diff
@@ -6,18 +6,19 @@
 
 class Shortcut:
 
-    def __init__(self, shortcut_name, exec_path, description="",
-                 icon_path="",
-                 desktop=False,
-                 start_menu=False, work_dir=None):
-        """
+    def __init__(self, shortcut_name: str, exec_path: str, description: str="",
+                 icon_path: str="",
+                 desktop: bool=False,
+                 start_menu: bool=False, work_dir: str=None):           
+        """Initialize a shortcut object.
```

Comment:
Please try to put docs not right behind quotes

Reply:
Oh why? 

https://peps.python.org/pep-0257/#multi-line-docstrings

Reply:
It is nicer I know it can be done both ways but i like the second way to do multi line comments